certifier_service/certlib/certlib_policy.go

```diff
@@ -290,15 +293,29 @@ func EncapsulateData(ek *certprotos.KeyMessage, alg string, data []byte, edm *ce
 		return false
 	}
 
+	pK := rsa.PrivateKey{}
+	PK := rsa.PublicKey{}
+	if !GetRsaKeysFromInternal(ek, &pK, &PK) {
+		fmt.Printf("Can't recover keys\n")
+		return false
+	}
+
+	label := "huh"
```

Comment:
What's this hard coded label? We have another one below.

Reply:
It's like a nonce, go crypto uses it to make sure the decrypt worked.